_posts/2018-10-25-gsoc-motion-planning-support.markdown

```diff
@@ -7,51 +7,51 @@ slug: gsoc-motion-planning-support
+One of this year's Google Summer of Code Projects (GSoC) was the project, “Adding motion planning support for motion planners in MoveIt”. This project, with coordination with PickNik Consulting, Open Source Robotics Foundation (OSRF), Dave Coleman, and Mark Moll, has been developed for the purpose of improving and extending the currently existing planners in MoveIt. An existing concept of planning adapters in MoveIt was used to have a pipeline of two planners running sequentially to provide robust solutions.
 
-Prior to summer 2018, MoveIt! only had a single stable motion planning library as its core motion planner, i.e. OMPL. As a part of this GSoC project, motion planning support was extended by making other motion planners (CHOMP) more stable and porting STOMP also to be a part of the MoveIt! motion planners family. The MoveIt! part of STOMP was ported from the [ros-industrial/industrial_moveit](https://github.com/ros-industrial/industrial_moveit) repository into the [moveit/motion planners](https://github.com/ros-planning/moveit/tree/kinetic-devel/moveit_planners). Furthermore, the benchmarking package was made to work with other motion planners in addition to OMPL and its [tutorials](https://ros-planning.github.io/moveit_tutorials/doc/benchmarking/benchmarking_tutorial.html) were added. This package provides user the option to visualize quantitatively different parameters (time taken, trajectory length, solved status, etc.) for different motion planners (OMPL, CHOMP, STOMP) in a common environment (with/without obstacles) and make a decision on which planner the user might want to use.
+Prior to summer 2018, MoveIt only had a single stable motion planning library as its core motion planner, i.e. OMPL. As a part of this GSoC project, motion planning support was extended by making other motion planners (CHOMP) more stable and porting STOMP also to be a part of the MoveIt motion planners family. The MoveIt part of STOMP was ported from the [ros-industrial/industrial_moveit](https://github.com/ros-industrial/industrial_moveit) repository into the [moveit/motion planners](https://github.com/ros-planning/moveit/tree/kinetic-devel/moveit_planners). Furthermore, the benchmarking package was made to work with other motion planners in addition to OMPL and its [tutorials](https://ros-planning.github.io/moveit_tutorials/doc/benchmarking/benchmarking_tutorial.html) were added. This package provides user the option to visualize quantitatively different parameters (time taken, trajectory length, solved status, etc.) for different motion planners (OMPL, CHOMP, STOMP) in a common environment (with/without obstacles) and make a decision on which planner the user might want to use.
```

Comment:
Odd how it decided to replace the entire paragraph

Reply:
This is unrelated to removing exclamation marks, but should we add an update to clarify that STOMP now lives at https://github.com/ros-industrial/stomp_ros.git ?

Reply:
For sure! Please do this in a separate PR